samples/matter/common/src/ota_util.h

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
It doesn't match the upstream. InitBasicOTARequestor is missing.

Reply:
Right, updated.